context.go

```
@@ -437,3 +437,16 @@ func (c *Context) GetReqHeader(key string) string {
 	return c.Request.Header
 }
+
+// 使用定义的errorHandle来处理error并结束当前handle
+func (c *Context) ErrorUseHandle(code int) {
+	if c.engine != nil && c.engine.errorHandle.handler != nil {
+		c.engine.errorHandle.handler(c, code)
+		c.Abort()
+		return
+	} else {
+		// Default error handling if no custom handler is set
+		c.String(code, http.StatusText(code))
```

golangci-lint (1.64.8)

printf: non-constant format string in call to (*github.com/WJQSERVER/touka.Context).String

(govet)